service/src/main/resources/application-azure.yml

```diff
@@ -2,7 +2,10 @@
 # in different environments
 workspace:
   azure:
-    tenantId: TBD
+    userTenantId: 0cb7a640-45a2-4ed6-be9f-63519f86e04b
```

Comment:
The user part of this should not be here. That is what lives in the AzureCloudContext.
I have similar configuration defined for use in testing in `application-azure-test.yml`.

As mentioned above, I have a parallel, but different implementation for the managed app coordinates.

Reply:
Yeah, just cleanup left to do after the rebase with your `AzureCloudContext`. Great to have this to separate the 'user' supplied config vs the config WSM is responsible for.